docs/compilation/modules.md

```diff
@@ -0,0 +1,57 @@
+# Modules
+
+{% hint style="warning" %}
+Modules are still experimental. They are only compatible with composition, that is, all function outputs can be used as inputs for every functions. The crypto-parameters used in this mode are pretty large, and will likely give slow execution time.
+{% endhint %}
+
+In some cases, it might be interesting to deploy a server that is able to execute different functions. With *Concrete*, it is possible to compile fhe _modules_, that can contain many different functions, at once. All the functions are compiled in a single step and can be [deployed with the same artifacts](../guides/deploy.md#deployment-of-modules). Here is an example:
```

Comment:
First sentence:
In some use cases, deploying a server that can execute different functions is useful.  *Concrete* can compile FHE _modules_ that contain many different functions to execute at once.